### ISSUE TITLE
Discover page has unequal margins

### ISSUE THREAD

**Issue by vazquez** [mozilla/webmaker-app#827]
The table cell margin (left and right) are not equal.

![screenshot 2015-01-28 12 58 33](https://cloud.githubusercontent.com/assets/1119821/5943480/688830be-a6ed-11e4-8657-ceef77e1cbe3.png)

**Comment by k88hudson** [comment]
also missing a border on the top item seems like

**Comment by gesa** [comment]
So I got the top border here, but I'm not totally certain where the margins are off. (I checked the text and icon margins from viewport/container, and they're the same)

**Comment by k88hudson** [comment]
yeah @gesa i think you are right, they are the same. due to optical human eye bs when aligning straight and curved objects, you generally need to make curved objects extrude slightly to make them appear aligned, which could be causing this issue.either that or we're looking at different browsers/apps. @vazquez thoughts?

**Comment by thisandagain** [comment]
Right on @k88hudson. This comment was in regards to the optical spacing rather than the actual calculation. A good example of how this type of optical alignment is compensated for is in typography:

![screen shot 2015-01-29 at 4 53 26 pm](https://cloud.githubusercontent.com/assets/747641/5969351/5f7eaaf8-a7d7-11e4-9950-4760362abd2f.png)

I *love* that we are getting into this level of detail. :heart:

**Comment by gesa** [comment]
Me too! (but to be 100% honest, I'm still not entirely certain what to do next on this ticket)

**Comment by thisandagain** [comment]
Hehe! Totally fair.

Because of the rounded corners on the segmented control above the app cell, I think we should keep the icon "bubble" on the right where it is. The easiest way to fix this optically is to move the text block on the left of the cell in by 1 or 2 pixels so it feels balanced / aligned with the control above the list.

**Comment by gesa** [comment]
👍

**Comment by flukeout** [comment]
It actually has to do with the angle of the cone receptors in the left eye specifically. If the offset curvature in the optics has sub-factoral relationship to the edge of the aligned object, you're going to have a bad time. Generally, a 2:3 translation ratio in the transverse axis of the alignment will do the trick. Give or take 2 pixels.

**Comment by vazquez** [comment]
What Luke said.

**Comment by k88hudson** [comment]
i prefer my technical terminology  i.e. *"due to optical human eye bs "* but if you want it in layman's terms sure we can go with that @flukeout

**Comment by davidascher** [comment]
Luke took the pedantry to 10. I've got to crank it to 11.

The problem isn't _optical_, it's _perceptual_.  It's the wetware after the optics and on the path to perception.

So there. ;-)

:spades: :hearts: :clubs: :diamonds:

**Comment by k88hudson** [comment]
Actually @davidascher, that is is 100% right on
